src/components/Card.js

```diff
@@ -42,11 +87,19 @@ class Card {
 
   getView() {
     this._element = this._getTemplate();
+    this._deleteButton = this._element.querySelector(".card__delete-button");
+    this._handleDeleteButton();
+    this._deleteButton.addEventListener("click", () =>{ this._handleDeleteClick()});
+    this._elementId = this._id;
     const cardImage = this._element.querySelector(".card__image");
     cardImage.src = this._link;
     cardImage.alt = `Photo of ${this._name}`;
     const cardTitle = this._element.querySelector(".card__title");
     cardTitle.textContent = this._name;
+    this._likeCounter = this._element.querySelector(".card__like-number");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**